fe/fe-core/src/main/java/com/starrocks/external/hive/HiveMetaClient.java

```diff
@@ -444,17 +444,21 @@ private double getLengthFromLiteral(LiteralExpr literalExpr, Type type) {
     private List<HdfsFileDesc> getHdfsFileDescs(String dirPath) throws Exception {
         URI uri = new URI(dirPath);
         FileSystem fileSystem = getFileSystem(uri);
-        FileStatus[] files = fileSystem.listStatus(new Path(uri.getPath()));
+        // fileSystem.listLocatedStatus is a batch api include fileSystem.listStatus
```

Comment:
fileSystem.listLocatedStatus is an api to list all statuses and block locations of the files in the given path in one operation. The performance is better than getting status and block location one by one.

Reply:
ok